src/Sylius/Bundle/UserBundle/spec/Security/UserPasswordHasherSpec.php

```diff
@@ -17,36 +17,40 @@
 use Sylius\Component\User\Model\CredentialsHolderInterface;
 use Sylius\Component\User\Security\UserPasswordEncoderInterface;
 use Sylius\Component\User\Security\UserPasswordHasherInterface;
+use Symfony\Component\HttpKernel\Kernel;
 use Symfony\Component\PasswordHasher\Hasher\PasswordHasherFactoryInterface;
 use Symfony\Component\PasswordHasher\PasswordHasherInterface;
 
-final class UserPasswordHasherSpec extends ObjectBehavior
-{
-    function let(PasswordHasherFactoryInterface $passwordHasherFactory): void
+if (Kernel::MAJOR_VERSION === 6) {
```

Comment:
We can ship this spec in a more elegant way IMO by throwing `\PhpSpec\Exception\Example\SkippingException` in `let` method if `Kernel::MAJOR_VERSION === 6`. And similar might be done in `UserPasswordEncoderSpec`.

Reply:
I see it's resolved but I don't see it in the code 🤔 @Rafikooo?

Reply:
We discussed it with @coldic3 via call. In brief, I couldn't use `EncoderFactoryInterface` in specs with Symfony 6 even though it definitely exists ( as deprecated ). Spec thinks that this class doesn't exist